converse.js

```diff
@@ -2186,7 +2186,9 @@
                         this.model.save();
                     }
                     this.setChatState(ACTIVE);
-                    this.scrollDown().focus();
+                    if (typeof callback === "object" && callback.focus === true) {
+                        this.scrollDown().focus();
+                    }
                 }.bind(this));
```

Comment:
You've moved both `scrollDown` and `focus` in to the if statement. We still want to scroll down, right? So above the `if` should be `this.scrollDown();` and inside should be `this.focus();`.